include/emp/math/Random.hpp

```diff
@@ -382,10 +389,10 @@ namespace emp {
       // Using Rejection Method and saving of initial exponential random variable
       double expRV2;
       while (1) {
-        expRV2 = -log(GetDouble());
+        expRV2 = -log(GetDoubleNonZero());
```

Comment:
Yeah log-ing zero will not turn out well. Curious, did this actually happen in production? That sounds like the most unlucky bug ever (i.e., getting EXACTLY zero)!!!

Reply:
😅 omg that sounds horrible to debug. Glad you caught this :)